packages/office-ui-fabric-react/src/components/Callout/examples/Callout.Directional.Example.tsx

```diff
@@ -27,6 +37,63 @@ const DIRECTION_OPTIONS = [
   { key: DirectionalHint.rightBottomEdge, text: 'Right Bottom Edge' }
 ];
 
+const theme = getTheme();
+const styles = mergeStyleSets({
+  buttonArea: {
+    verticalAlign: 'top',
```

Comment:
Was there a reason for getting rid of some of the styles from `.ms-CalloutExample-buttonArea`? Specifically:
```
    min-width: 130px;
    margin: 0 100px;
    height: 32px;
```

Reply:
Re-added! I missed those styles somehow